src/som/vm/Universe.java

```diff
@@ -53,21 +58,40 @@
 import som.vmobjects.SSymbol;
 
 
-public class Universe {
+public final class Universe {
 
-  public static void main(final String[] arguments) {
-    // Create Universe
-    Universe u = new Universe();
+  static final Layout LAYOUT = Layout.createLayout();
```

Comment:
Hm, I wouldn't start with things like object layouts just yet.
Let's see that we get the basic execution mechanisms ready for the partial evaluation.